pkg/cli/cve_cmd.go

```diff
@@ -148,13 +150,40 @@ func searchCve(searchConfig searchConfig) error {
 	}
 
 	for _, searcher := range searchers {
-		found, err := searcher.search(searchConfig)
-		if found {
-			if err != nil {
+		// there can be CVE DB readyness issues on the server side
+		// we need a retry mechanism for that specific type of errors
+		maxAttempts := 20
+
+		for i := 0; i < maxAttempts; i++ {
+			found, err := searcher.search(searchConfig)
+			if !found {
+				// searcher does not support this searchConfig
+				// exit the attempts loop and try a different searcher
+				break
+			}
+
+			if err == nil {
+				// searcher matcher search config and results are already printed
+				return nil
+			}
+
+			if i+1 >= maxAttempts {
+				// searcher matches search config but there are errors
+				// this is the last attempt and we cannot retry
 				return err
 			}
 
-			return nil
+			if strings.Contains(err.Error(), zotErrors.ErrCVEDBNotFound.Error()) {
```

Comment:
Not sure unwrapping the error would work client side if the wrapping was done server side. It's not the same object, I suspect that is the reason.
I tried in the previous implementation when I was wrapping 2 errors, and it didn't work. I left the Contains() in case we want to add a new string explaining which store / path has the issue, but sending that information to zli seems like a potential security issue to me.